### ISSUE TITLE
[ART] LinearGradient on Android not working

### ISSUE THREAD

**Issue by antoxi4** [facebook/react-native#5200]
When i trying fill shape with LinearGradient my shape fills with black color.

**Comment by satya164** [comment]
@antoxi4 React Native doesn't support LinearGradient AFAIK. How are you trying to use it?

**Comment by browniefed** [comment]
@satya164 It is supported ;) https://rnplay.org/apps/5tzUPQ

**Comment by satya164** [comment]
Aah. ART :D

ART was very recently released for Android though. If you have any issues with it, please report with code sample. @antoxi4

**Comment by antoxi4** [comment]
This is my code)It working on IOS but not working on Android)
```javascript
<Shape d={HEART} fill={new LinearGradient({'.5': '#F48FB1', '1': '#2196F3'},"128","0","235.3903723704666","58.34866891341897")}/>
```

**Comment by mkonicek** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with ProductPains! Check out its new home: https://productpains.com/post/react-native/art-lineargradient-on-android-not-working

ProductPains helps the community prioritize the most important issues thanks to its voting feature.
It is easy to use - just login with GitHub.

Also, if this issue is a bug, please consider sending a PR with a fix.
We're a small team and rely on the community for bug fixes of issues that don't affect fb apps.

**Comment by mikelambert** [comment]
This appears like it will be fixed in RN 0.46: https://github.com/facebook/react-native/commit/36405d13ffc5ec25a2e0cf5d811d0fd767b0dfa5
